### PR TITLE
AnneJu step 5: add README of models and web image drafts

### PR DESCRIPTION
**Pull request by anneju**
- Step4: Imagine the application that you want to create
- Step5: Draft DB columns

---
ref: [instruction](https://github.com/Fablic/training/tree/use_docker)

### CODE REVIEW

File: myapp/README.md [hunk 1]
```diff
@@ -11,7 +11,51 @@ Things you may want to cover:
...
+
 * Database creation
+  - Model draft picture: https://officerakuten-my.sharepoint.com/:i:/g/personal/anne_ju_rakuten_com/EZ6wLQ423BlBl7cb5QXJw1ABw-11UP8nTCJW0rL_6qr7Ww?e=vSdTOz
+
+  - User
+    - id integer
+    - name string
+    - email string
+    - passward string
+    - is_damin boolean
```
Comment (shunshunsuke) [inline myapp/README.md]: `is_admin` is collect?

File: myapp/README.md [hunk 2]
```diff
@@ -11,7 +11,51 @@ Things you may want to cover:
...
+  - https://officerakuten-my.sharepoint.com/:p:/g/personal/anne_ju_rakuten_com/EVCW0ukKm5BMqsaFkDJOF_EBfOlfXEAaGSfkLPTvwOx7nw?e=WMm8rQ
+
 * Database creation
+  - Model draft picture: https://officerakuten-my.sharepoint.com/:i:/g/personal/anne_ju_rakuten_com/EZ6wLQ423BlBl7cb5QXJw1ABw-11UP8nTCJW0rL_6qr7Ww?e=vSdTOz
+
+  - User
+    - id integer
+    - name string
+    - email string
+    - passward string
```
Comment (shunshunsuke) [inline myapp/README.md]: If `bcrypt` is used, the name must be `password_digest`.

**Review by shunshunsuke — APPROVED** [review]
LGTM!
Please review a few comments.
LGTM

File: myapp/README.md (hunk 1 above)
Comment (anneju) [reply myapp/README.md]: yeah, the typo~~ 🙈

File: myapp/README.md (hunk 2 above)
Comment (anneju) [reply myapp/README.md]: correct~~ 🙆🏻‍♀️

**Comment by anneju** [comment]
@shunshunsuke Should I remove your approval and let you review again with the new fix commit? Or just merge it?

**Comment by shunshunsuke** [comment]
@anneju You can merge it!

**Pull request merged**